Tools/WebKitTestRunner/TestController.h

```diff
@@ -45,6 +45,10 @@
 #include "InstanceMethodSwizzler.h"
 #endif
 
+#if PLATFORM(GTK) || PLATFORM(WPE)
+#include "WKContextConfigurationGlib.h"
+#endif
+
```

Comment:
Why do we need this in the header?

Reply:
`TestController` calls `WKContextConfigurationSetOverrideDefaultCairoFontOptionsForTesting()` function which is defined in a separate GLIB-specific header `WKContextConfigurationGlib.h`. That's why this include is needed.

Reply:
I could move it to `WKContextConfigurationRef.h` and surround it with `#if PLATFORM(GTK) || PLATFORM(WPE)`. I'm not sure which approach is better. I see that PlayStation port also adds some API using a separate header and did the same.

Reply:
Can't it be moved to TestController.cpp?

Reply:
I've moved these GLIB-specific functions to `WKContextConfigurationRef.h` to the other `WKContextConfiguration*` functions. No additional headers is needed anymore.

Reply:
That didn't work. Moving it back to a separate header. And, as you suggested, include it in `TestController.cpp`.